src/libs/actions/Policy/Policy.ts

```diff
@@ -2930,6 +2943,55 @@ function enableDistanceRequestTax(policyID: string, customUnitName: string, cust
     API.write(WRITE_COMMANDS.ENABLE_DISTANCE_REQUEST_TAX, params, onyxData);
 }
 
+function enablePolicyInvoices(policyID: string, enabled: boolean) {
```

Comment:
```suggestion
function enablePolicyInvoicing(policyID: string, enabled: boolean) {
```

to match the api command

Reply:
Done - 5bdf617.